landscape.yml

```diff
@@ -1029,6 +1029,26 @@ landscape:
               wiki: https://wiki.hyperledger.org/display/fabric
               chat: https://discord.com/channels/905194001349627914/905205541326975046
               documentation: https://hyperledger-fabric.readthedocs.io/en/latest/
+          - item:
+            name: Hiero
+            second_path:
+              - Distributed Ledgers
+            description: Hiero is the open source DLT codebase used to build the Hedera network; including the hashgraph consensus algorithm and all core services, tooling and libraries. Hiero supports the development of a wide range of decentralized applications and core platform features tailored for the Hedera network, including wallets, exchanges, explorers, bridges, SDKs, private ledgers, and advanced cryptographic solutions.
+            homepage_url: https://www.lfdecentralizedtrust.org/projects/hiero
+            project: incubting
+            repo_url: https://github.com/hiero-ledger/hiero
+            additional_repos:
+              - repo_url: https://github.com/hiero-ledger/awesome-contributions
+              - repo_url: https://github.com/hiero-ledger/governance
+              - repo_url: https://github.com/hiero-ledger/hiero-gradle-conventions
+              - repo_url: https://github.com/hiero-ledger/hiero-sdk-cpp
+              - repo_url: https://github.com/hiero-ledger/hiero-sdk-tck
+              - repo_url: https://github.com/hiero-ledger/hiero-website
+              - repo_url: https://github.com/hiero-ledger/tsc
+            logo: heiro_stroke.svg
```

Comment:
This is the one I mean @ryjones. The logo this PR adds is named `hiero_stroke.svg`, and they need to match.